test/helpers/publishers_helper_test.rb

```diff
@@ -127,6 +127,43 @@ def initialize(wallet_json:)
     assert_equal "Unavailable", publisher_humanize_balance(publisher, "USD")
   end
 
+  test "publisher_possible_currencies should an array of possible currencies" do
```

Comment:
Missing a word (be)? "publisher_possible_currencies should an array of possible currencies" -> "publisher_possible_currencies should be an array of possible currencies"